packages/core/src/components/MonthPicker/MonthPicker.example.jsx

```diff
@@ -24,6 +24,13 @@ ReactDOM.render(
       label="Select the months you were in cryosleep during 2046"
       disabledMonths={[7, 8, 9, 10, 11, 12]}
     />
+    <MonthPicker
+      name="cryosleep-months"
+      label="Select the months you were in cryosleep during 2046"
+      disabledMonths={[7, 8, 9, 10, 11, 12]}
+      inversed
```

Comment:
This is looking pretty funky, do we need this example?

![image](https://user-images.githubusercontent.com/371943/35584258-7903b6b6-05c2-11e8-8209-e4a22c6a4f2d.png)